src/components/common/trends-add/trends-add.tmpl.html

```diff
@@ -0,0 +1,34 @@
+<div id="trends-add">
+  <div class="overlay" />
+  <div class="message" >
+    <div class="title">Add data from historical trend</div>
+
+    <div class="form text-left">
+      <label>Adjustment rate:</label><br>
+      <input style="width: 45px;" type="number" value="0" min="0" step="1" ng-model="$ctrl.trend.rate"/>%<br>
+
+      <label>Trend applied:</label><br>
+      <select ng-model="$ctrl.trend.period">
+        <option>Once</option>
+        <option>Daily</option>
+        <option>Weekly</option>
+        <option>Monthly</option>
+        <option>Annually</option>
+      </select>
+
+      <fieldset class="btn-group" ng-disbaled="$ctrl.isPeriodDisabled()">
+        <input type="radio" ng-model="$ctrl.trend.untilDate" name="until" checked="true" ng-value=-1> Forever<br>
```

Comment:
The `ng-value` is missing quotations around the value on this input and the input below this line. Are you sure its working / needed?